src/main/scala/mesosphere/marathon/core/deployment/ScalingProposition.scala

```diff
@@ -2,50 +2,51 @@ package mesosphere.marathon
 package core.deployment
 
 import mesosphere.marathon.core.condition.Condition
-import mesosphere.marathon.core.instance.Instance
+import mesosphere.marathon.core.condition.Condition.UnreachableInactive
+import mesosphere.marathon.core.instance.{Goal, Instance}
 import mesosphere.marathon.state.{KillSelection, Timestamp}
 
 case class ScalingProposition(tasksToKill: Option[Seq[Instance]], tasksToStart: Option[Int])
 
 object ScalingProposition {
 
   def propose(
-    runningTasks: Seq[Instance],
-    toKill: Option[Seq[Instance]],
+    instances: Seq[Instance],
+    toDecommission: Option[Seq[Instance]],
```

Comment:
Why not
```suggestion
    toDecommission: Seq[Instance],
```

Reply:
What do you think?

Reply:
Oooh I missed this! I thought I did both!